### PR TITLE
added bootstrap back to custom scss and reran css

### PR DESCRIPTION
**Pull request by nwebb-roux**
@PhilipMathieu Would you check this out? When I merged your CSS PR, the page lost its styling. As far as I can tell, we still need to bring in the core bootstrap scss directory in custom.scss, but I may be misunderstanding your goal with your rewrite of custom.scss.

For now, this PR does seem to fix the page styling, though, so it may be worth it as a quick fix.

### CODE REVIEW

**Comment by PhilipMathieu** [comment]
I'm not home right now so I can't easily investigate, but I'm going to go ahead and approve this request just so I'm not holding anyone else up. I think I know what I did wrong but it's too hard to figure it out on my phone.

**Review by PhilipMathieu — APPROVED** [review]

**Comment by PhilipMathieu** [comment]
The problem was basically just that I had completely misread the bootstrap docs and thought I was using the correct template but hadn't read the fine print. I pushed a commit to eliminate redundant imports; otherwise, this should be good to go as soon as someone else reviews it.

File: docs/css/custom.css [hunk 1]
```diff
@@ -1,4 +1,10 @@
 @charset "UTF-8";
```
Comment (bergsalex) [inline docs/css/custom.css:1]: Is this file auto generated from the scss file?

There's so much css here!

**Review by bergsalex — APPROVED** [review]

**Comment by PhilipMathieu** [comment]
Yes - it's basically all of bootstrap plus our overrides.

Since we didn't actually customize anything within bootstrap (just added a custom class) it's unnecessary for this project, but could be helpful if we want to make any other visual tweaks. For instance, if you import bootstrap this way you can override default fonts and colors, we just didn't actually do that.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**